hips/tiles/tile.py

```diff
@@ -0,0 +1,117 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import urllib.request
+from io import BytesIO
+
+import numpy as np
+from PIL import Image
+from astropy.io import fits
+from astropy.io.fits.header import Header
+
+from .tile_meta import HipsTileMeta
+from ..utils.tile import tile_path
+
+__all__ = [
+    'HipsTile',
+]
+
+__doctest_skip__ = ['HipsTile']
+
+
+class HipsTile:
+    """HiPS tile container.
+
+    This class provides methods for fetching, reading, and writing a HiPS tile.
+
+    Parameters
+    ----------
+    hips_tile_meta : `HipsTileMeta`
+        Metadata of HiPS tile
+    format : `str`
+        Format of HiPS tile
+    data : `np.ndarray`
```

Comment:
Please always check the HTML docs locally. In this case you'll find that the link is broken and you have to put `~numpy.ndarray` instead of `np.ndarray`.